### PR TITLE
issue/41 Move settings to a dedicated tab

### PR DESCRIPTION
**Pull request by dbeja**
Issue #41 

**Housekeeping**

* [x] Issue number has been linked below the description
* [x] Branch name follows the `issue/###` or `release/###` format
* [x] PR title is prefixed with the branch name (issue/### - Title)

**Proposed changes**

* [x] Code has been tested with no errors
* [x] Code meets stated minimum requirements
* [x] Code should (usually) not contain unrelated changes
* [x] Code style (including docs) and organization is generally consistent with the rest of the codebase
* [x] Before and after screenshots have been posted to document visual changes
* [x] Clear steps for testing the PR have been posted (as needed)

**Before changes**

imagem

**After changes**

imagem

### CODE REVIEW

**Review by amdrew — APPROVED** [review]
Works great for me 👍 

Screen Shot 2021-01-11 at 1 23 35 PM

File: includes/class-admin.php [hunk 1]
```diff
@@ -4,58 +4,64 @@ class AffiliateWP_Checkout_Referrals_Admin {
...
+				'type' => 'checkbox'
+			),
+			'checkout_referrals_affiliate_display' => array(
+				'name' => __( 'Affiliate Display', 'affiliatewp-checkout-referrals' ),
+				'desc' => __( 'How the affiliates will be displayed at checkout', 'affiliatewp-checkout-referrals' ),
+				'type' => 'radio',
+				'options' => array(
+					'user_nicename' => 'User Nicename',
+					'display_name' => 'Display Name',
+					'nickname'     => 'Nickname'
```
Comment (DrewAPicture) [inline includes/class-admin.php:58]: The option values need to be translatable.

File: includes/class-admin.php [hunk 2]
```diff
@@ -4,58 +4,64 @@ class AffiliateWP_Checkout_Referrals_Admin {
...
+				'std'  => __( 'Who should be awarded commission for this purchase?', 'affiliatewp-checkout-referrals' )
 			),
-			'std' => 'user_nicename'
+			'checkout_referrals_affiliate_selection' => array(
+				'name' => __( 'Affiliate Selection Method', 'affiliatewp-checkout-referrals' ),
+				'desc' => __( 'Choose how the customer will select the affiliate to award commission to.', 'affiliatewp-checkout-referrals' ),
+				'type' => 'select',
+				'options' => array(
+					'select_menu' => __( 'Customer selects affiliate from list', 'affiliatewp-checkout-referrals'),
+					'input'       => __( 'Customer enters affiliate\'s ID or username', 'affiliatewp-checkout-referrals'),
```
Comment (DrewAPicture) [inline includes/class-admin.php:43]: The apostrophe in this sentence should be encoded. These various grammar nitpicks are not your fault obviously, but we should fix them while we're here.
```suggestion
					'input'       => __( 'Customer enters affiliate&#8217;s ID or username', 'affiliatewp-checkout-referrals'),
```

File: includes/class-admin.php [hunk 3]
```diff
@@ -4,58 +4,64 @@ class AffiliateWP_Checkout_Referrals_Admin {
...
+				)
+			),
+			'checkout_referrals_require_affiliate' => array(
+				'name' => __( 'Require Affiliate Selection', 'affiliatewp-checkout-referrals' ),
+				'desc' => __( 'Customer must select an affiliate to award the referral to', 'affiliatewp-checkout-referrals' ),
```
Comment (DrewAPicture) [inline includes/class-admin.php:48]: There should be a period at the end of this sentence.
```suggestion
				'desc' => __( 'Customer must select an affiliate to award the referral to.', 'affiliatewp-checkout-referrals' ),
```

File: includes/class-admin.php [hunk 4]
```diff
@@ -4,58 +4,64 @@ class AffiliateWP_Checkout_Referrals_Admin {
...
+				'type' => 'checkbox'
+			),
+			'checkout_referrals_affiliate_display' => array(
+				'name' => __( 'Affiliate Display', 'affiliatewp-checkout-referrals' ),
+				'desc' => __( 'How the affiliates will be displayed at checkout', 'affiliatewp-checkout-referrals' ),
```
Comment (DrewAPicture) [inline includes/class-admin.php:53]: And a period at the end of this sentence also.
```suggestion
				'desc' => __( 'How the affiliates will be displayed at checkout.', 'affiliatewp-checkout-referrals' ),
```

**Review by DrewAPicture — CHANGES_REQUESTED** [review]
Few string-related fixes needed, otherwise it looks good.

File: includes/class-admin.php (hunk 2 above)
Comment (dbeja) [reply includes/class-admin.php]: Yes, I agree!

**Review by DrewAPicture — APPROVED** [review]
Looks good.

**Pull request merged**